cogs/Alerts.py

```diff
@@ -152,46 +169,48 @@ async def user_alerts() -> None:
             If a message contains a keyword, send a DM to the user who added the keyword
             :return: The message.content is being returned.
             """
-            if (
-                message.author.bot
-                or message.channel not in message.author.guild.channels
-                or not message.author.guild.get_member(self.bot.user.id)
-            ):
+            assert (
+                self.bot.user is not None
+            ), "on_message only fires when the bot is already logged in"
+
+            # do not process alerts for bot messages or in DMs
+            if message.author.bot or isinstance(message.author, discord.User):
                 return
 
-            # Ignore messages that do not contain a keyword.
-            c = self.db.cursor()
-            c.execute("SELECT * FROM alert")
-            keywords = c.fetchall()
-            if not any(
-                re.search(keyword[2], message.content, re.IGNORECASE)
-                for keyword in keywords
-            ):
```

Comment:
Got rid of this because it's not doing anything useful as far as I can tell. If none of the keywords are found, then the next for loop won't do anything anyway - this check just doubles the amount of computation that the bot needs to do.